scripts/windows/test/Setup-Env.ps1

```diff
@@ -68,6 +68,11 @@ $IoTEdgedArtifactFolder = Join-Path $E2ETestFolder "artifacts\iotedged-windows"
 $InstallationScriptPath = Join-Path $E2ETestFolder "artifacts\core-windows\scripts\windows\setup\IotEdgeSecurityDaemon.ps1"
 Invoke-Expression $InstallationScriptPath
 
+If ((Test-Path "C:\data\edgehub"))
```

Comment:
I will make var for this path when I make the same changes to master, then port it back. We likely don't have time to rebuild the images.